### PR TITLE
[WIP] smarter revised robots.txt

### PR DESCRIPTION
**Pull request by dwaynebailey**
* Allow the whole site by default
* Disallow any of the editor and export views
* Allow access to the browse views, they're not expensive anymore
* Allow access to static pages, they should mostly be static content

### CODE REVIEW

File: pootle/apps/pootle_app/views/index/robots.py [hunk 1]
```diff
@@ -15,8 +15,11 @@ def view(request):
     """generates the robots.txt file"""
     langcodes = [language.code for language in Language.objects.iterator()]
     content = "User-agent: *\n"
-    for path in ["accounts", "projects", "unit", "xhr"]:
+    content += "Allow: /\n"
+    for path in ["projects", "projects/*", "pages"]:
+        content += "Allow: /%s/\n" % path
+    for path in ["*/translate", "*/*/translate",
```
Comment (unho) [inline pootle/apps/pootle_app/views/index/robots.py]: Is it possible to use wildcards in robots.txt?

File: pootle/apps/pootle_app/views/index/robots.py [hunk 2]
```diff
@@ -15,8 +15,11 @@ def view(request):
     """generates the robots.txt file"""
     langcodes = [language.code for language in Language.objects.iterator()]
```
Comment (unho) [inline pootle/apps/pootle_app/views/index/robots.py]: This is no longer used?

Comment (dwaynebailey) [inline pootle/apps/pootle_app/views/index/robots.py]: Removed

**Comment by dwaynebailey** [comment]
FYI This is what the new code generates:

```
User-agent: *
Allow: /
Allow: /projects/
Allow: /projects/*/
Allow: /pages/
Disallow: /*/translate/
Disallow: /*/*/translate/
Disallow: /*/export-view/
Disallow: /*/*/export-view/
Disallow: /user/*/stats/
Disallow: /accounts/
Disallow: /unit/
Disallow: /xhr/
```

**Comment by iafan** [comment]
Do we need _code_ to generate this? It could be a static (and overidable) file, since it doesn't contain any dynamic stuff like it did before (list of languages).

**Comment by dwaynebailey** [comment]
@iafan nope, we don't need code.  My figuring was to just get the autogenerated part right here and then retire.

**Comment by dwaynebailey** [comment]
@unho @iafan sorry I managed to mess up my branches and opened a new PR https://github.com/translate/pootle/pull/5008

**Pull request closed without merging**